tests/unit-pass/Makefile

```diff
@@ -1,18 +1,12 @@
 TUS = $(wildcard ./*.c) $(wildcard ./*.C)
 EXCLUDES=
-OS_TUS = $(wildcard ./j*.c) $(wildcard ./*.C)
-OS_EXCLUDES = \
-	./stdio-scanf11.c ./stdio-scanf12.c ./stdio-scanf15.c \
-	./stdio-bug3.c ./stdio-bug5.c ./stdio-bug6.c ./stdio-bug7.c ./stdio-bug12.c ./stdio-bug18.c \
-	./stdio-bug19.c ./stdio-bug20.c \
-	./stdio-fwrite.c ./stdio-temptest.c ./compcertTSO-2.c \
-	./stdio-ferror.c ./stdio-fgets.c ./stdio-fseek.c ./stdio-scanf-flush.c ./stdio-ungetc.c \
-	./stdio-sscanf.c ./stdio-fileno.c \
-	./inst-check.c ./pthreads-starve.c ./fopen-enoent.c \
-	./oldstyle-main.c ./struct-ini-1.c ./j016.c ./include-time.C \
-	./new23.C ./new41.C
+OS_STDIO_EXCLUDES = $(wildcard ./stdio-*.c)
+OS_EXCLUDES = $(OS_STDIO_EXCLUDES) \
+	./pthreads-starve.c ./fopen-enoent.c ./oldstyle-main.c ./j016.c \
+	./struct-ini-1.c ./20051113-1.c ./20050502-1.c ./string-opt-5.c ./inst-check.c \
```

Comment:
why did this list of tests change?

Reply:
Those were all excluded before -- previously it was only running the `j*.c` tests. I just enabled a bunch (~800) more tests.

Some tests fail on the OS semantics (like the `stdio-*.c` ones) because of missing stdlib support, but others fail for reasons I'm not clear about. I've fixed several issues here, but this needs to be revisited.